python/lsst/dax/apdb/__init__.py

```diff
@@ -21,11 +21,8 @@
 
 from .apdb import *
 from .apdbCassandra import *
-from .apdbCassandraSchema import *
 from .apdbMetadata import *
-from .apdbSchema import *
+from .apdbSchema import ApdbTables
 from .apdbSql import *
-from .apdbSqlSchema import *
```

Comment:
I would prefer if the config classes, at least, were still documented -- https://pipelines.lsst.io/v/daily/py-api/lsst.dax.apdb.ApdbCassandraConfig.html#lsst.dax.apdb.ApdbCassandraConfig, https://pipelines.lsst.io/v/daily/py-api/lsst.dax.apdb.ApdbConfig.html#lsst.dax.apdb.ApdbConfig, and https://pipelines.lsst.io/v/daily/py-api/lsst.dax.apdb.ApdbSqlConfig.html#lsst.dax.apdb.ApdbSqlConfig are by far the easiest sources of information for what the options are called and how they can be configured.

Reply:
Sorry, I misunderstood what I did 🙂 Config classes are still documented, as they come from `apdbSql` and `apdbCassandra` and those are not removed. Only some internal classes which implement schema management and old factory classes were removed.